Mathlib/Util/TransImports.lean

```diff
@@ -0,0 +1,31 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+
+import Mathlib.Init
```

Comment:
Should this file be imported into Mathlib.Init instead? So that it's available in any Mathlib file.

Reply:
I imported it in `Mathlib.Tactic.Common`.  I think that there is the intention to make the imports of `Mathlib.Init` somewhat restricted : `Mathlib.Tactic.Common` is still imported in a large part of Mathlib anyway.